### PR TITLE
Steps 1 and 2

### PR DESCRIPTION
**Pull request by jchavarri**
Adds steps 1 and 2:

- Step 1: Setting up the editor environment
- Step 2: Our first ReasonReact component

Will put the code modifications in a separate branch `step-2` after merging.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,25 +89,106 @@ opam init --reinit
...
-After the above are installed, proceed to run `npm run init` to download the
-workshop required packages from both npm and opam repositories.
+```sh
+git clone https://github.com/ahrefs/react-alicante-workshop
+cd react-alicante-workshop
+```
+
+### 0.4: Install all dependencies
+
+Proceed to run `npm run init` in order to download the workshop required
```
Comment (feihong) [inline README.md:103]: Maybe add a note saying how long to expect it to run? I haven't timed it lately, but I think it's often over 10 minutes.

File: README.md [hunk 2]
```diff
@@ -64,25 +89,106 @@ opam init --reinit
...
+We only did a small refactor, so the build tab running `npm run build` should
+show no errors, and the local page under http://localhost:8080/ should look like
+it did before starting this step.
+
 ## Project layout
 
 The following is a high level view of your project and application. Many of
 these files will contain additional comments, explanations, examples, and help
-for learning and getting started with OCaml, ReasonML and Melange.
+for learning and getting started with OCaml, Reason and Melange.
```
Comment (feihong) [inline README.md]:
```suggestion
for learning and getting started with OCaml, Reason, and Melange.
```

**Review by feihong — APPROVED** [review]
Looks great so far!

File: README.md (hunk 1 above)
Comment (jchavarri) [reply README.md:103]: Good point, added something about this in https://github.com/ahrefs/react-alicante-workshop/pull/2/commits/0c972b92a02750d7d3260af6fa07d8ac28970305

**Pull request merged**